src/doc/gui_manual/Preferences/File_Locations.rst

```diff
@@ -0,0 +1,297 @@
+* Written: Each time VisIt_ is started, the integer value in the associated
+  state tracking file is updated.
+* Read: Each time VisIt_ is started, the value in the associated state tracking
+  file is read.
+* Format: ASCII text
+
+Crash Recovery Files
+""""""""""""""""""""
+* Location and file name(s): ``VUSER_HOME/crash_recovery.$pid.session`` and
+  ``VUSER_HOME/crash_recovery.$pid.session.gui`` where ``$pid`` is the process
+  id of the VisIt_ viewer component.
+* Purpose: Hold the most recently saved last good state of VisIt_ prior
+  to a crash. Disabled if the preference
+  ``Periodically save a crash recovery file`` is unchecked in the
+  **Preferences Window**.
+* Written: Periodically from VisIt_ automatically.
+* Read: When user starts VisIt_ and answers ``yes`` when queried whether to
+  start up from the most recent crash recovery file or when user explicitly
+  specifies the crash recovery file as an argument to the ``-sessionfile``
+  command-line :ref:`startup option <StartupOptions>`.
+* Format: ASCII `XML <https://en.wikipedia.org/wiki/XML>`_, same as any
+  other VisIt_ :ref:`session files <Session files>`.
+
+Files In Other Locations
+~~~~~~~~~~~~~~~~~~~~~~~~
+
+There are several other kinds of files VisIt_ reads and writes to locations
+other than ``VUSER_HOME``. These are breifly described here.
+
+VisIt Debug Log (``.vlog``) Files
+"""""""""""""""""""""""""""""""""
+* Location and file name(s): On Windows, these files are always located in
+  ``VUSER_HOME``. On Unix/OSX the location of these files depends on whether
+  VisIt_ is being run in :ref:`client/server mode <Client-Server Mode>`.
+  When running client/server, some logs are written on the client and some on
+  the server. The logs on the client are written to whatever the current working
+  directory was when VisIt_ was started. If started from an icon, this is most
+  likely the the user's home directory. If started from a command-line, it is
+  whatever the shell's current working directory for that command-line was. On
+  the server, the logs are written to the user's login (home) directory. In a
+  typical client/server scenario, the user gets gui and viewer logs locally in
+  the current working directory and mdserver and engine logs on the remote
+  system in their login (home) directory. In a purely local scenario, all logs
+  are written to the current working directory.
+
+  The names of the log files are of the form
+  ``<letter>.<component-name>.<mpi-rank-or-$pid>.<debug-level>.vlog`` where
+  ``<letter>`` is one of ``A`` through ``E``, ``<component-name>`` is one of
+  ``gui``, ``mdserver``, ``viewer``, ``engine_ser``, ``engine_par``,
+  ``<mpi-rank-or-$pid>`` is the MPI rank for a prallel engine (``engine_par``)
+  or, optionally if ``-pid`` is given as a command-line
+  :ref:`startup option <StartupOptions>`) the component's process id,
+  and ``<debug-level>`` is the integer argument for the ``-debug``
```

Comment:
Sorry Mark. Naming is different on Windows:
parallel:
<component-name>.exe.<rank>.<pid>.<debug-level>.vlog
serial
<component-name>.exe.<pid>.<debug-level>.vlog
There is never a letter at the beginning, and pid is always used.